### ISSUE TITLE
Archi closes when trying to access the Help menu item (Top level)

### ISSUE THREAD

**Issue by duskoknez** [archimatetool/archi#927]
### Version of Archi

5.0.1

### Archi Plug-ins

coArchi, jArchi, Lightbox for Archi

### Operating System

OSC Cataline 10.15.7

### Expected Behaviour

Should be able to open the help menu

### Actual Behaviour

Archi closes/crashes and a message that a java 6 runtime is required with a link to the download page.  Catalina doesn't support this version of the java runtime
### Steps to Reproduce the Behaviour
1. Open Archi
2.  Try to access the Help menu - Archi closes and a window pops up asking for Java 6 runtime
3.
![archi error](https://user-images.githubusercontent.com/129900362/229910937-e04f92d9-fe8d-4f37-b5e7-ef1f6d1b954e.png)

**Comment by Phillipus** [comment]
This is very strange. Archi ships with its own Java runtime so doesn't need another one. I don't have a copy of Catalina but tested on Monterey, Ventura and even an old copy of Sierra.

Is it possible to test on another machine or macOS version? As this issue has not been reported before I suspect this must be something to do with your setup. Perhaps even try downloading Archi again?

**Comment by duskoknez** [comment]
When i originally started using version 4, i didn't see this issue.  it started happening recently, but i can' t pin point exactly when.  I will try downloading Archi again.  Since the plugin management menu item is under help, it's clear that i was able to access it at one point earlier.

**Comment by Phillipus** [comment]
This is going to be hard to figure out. If there were other reports of this issue we could say it was maybe a bug, but this hasn't been reported before.

Maybe clear some caches by deleting these folders:

`/Users/your_user_name/Library/Application Support/Archi/config`

`/Users/your_user_name/Library/Caches/com.archimatetool.editor`

`/Users/your_user_name/Library/Saved Application State/com.archimatetool.editor.savedState`

**Comment by duskoknez** [comment]
That did it.. awesome.. thanks

**Comment by Phillipus** [comment]
OK. Thanks for reporting this bug, strange that no-one reported it before.

Here's the analysis:

Before Archi 5 I always set the path to Java as `../jre/lib/libjli.dylib`. When I made changes to Archi 5 I simplified this to `../jre` and it works OK when tested on Monterey and Ventura.

Where it doesn't work is on earlier versions of macOS....unless you also have Java installed in the system. I also have Java installed in the system so this worked for me when I tried it on Sierra. Once I uninstalled Java on Sierra I got the same problem as you.

So I'm changing this back in tomorrow's build of Archi 5.0.2.

Good job I still had an older copy of macOS.
